examples/gameroom/cli/Cargo.toml

```diff
@@ -0,0 +1,40 @@
+# Copyright 2018-2020 Cargill Incorporated
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[package]
+name = "gameroom-cli"
+version = "0.1.0"
+authors = ["Cargill Incorporated"]
+edition = "2018"
+license = "Apache-2.0"
+
+[[bin]]
+name = "gameroom"
+path = "src/main.rs"
+
+[dependencies]
+clap = "2"
+flexi_logger = "0.14"
+log = "0.4"
+
+[package.metadata.deb]
+maintainer = "The Splinter Team"
+depends = "$auto"
+
+[features]
```

Comment:
This should go before the package.metadata.db section

Reply:
Done